docs/subnets/register-validate-mine.md

```diff
@@ -79,7 +93,7 @@ See [Becoming a delegate](../staking-and-delegation/delegation.md#becoming-a-del
 
 ### Validator permit
 
-Only the largest 64 subnet validators, in terms of stake, on any particular subnet are considered to have **validator permit**. Only the subnet validators with permit are considered active in a subnet. 
+Only the 64 most significant subnet validators, in terms of stake, on any particular subnet are considered to have **a validator permit**. Only the subnet validators with permits are considered active in a subnet.
```

Comment:
Actually the original description here is ambiguous. Let's clarify it like this:

"Only the top 64 validators, when ranked by their stake amount, in any particular subnet, are considered to have **a validator permit**. Only these top 64 subnet validators with permits are considered active in the subnet."